ROADMAP.md

```diff
@@ -2,6 +2,26 @@
 
 This document defines a high level roadmap for OpenYurt development and upcoming releases. Community and contributor involvement is vital for successfully implementing all desired items for each release. We hope that the items listed below will inspire further engagement from the community to keep OpenYurt progressing and shipping exciting and valuable features.
 
+## 2025 H1
+- Support Kubernetes version 1.32.
+- Upgrade node pools to v1beta2, Support multiplexer list/watch requests at the node pool level.
+- Merge Raven into the OpenYurt main repository.
+- Develop a tool(or a controller in yurt-manager) to install yurthub component in a standard K8s cluster. 
+- Reconstruct the framework of controllers and webhooks of yurt-manager.
+
+## 2025 H2
+- Support LoadBalancer Services across multiple node pools.
+- Support request multiplexing for CRD resources at the node pool level.
```

Comment:
Support request multiplexing for CRD resources at the node pool level. --> Support aggregate list/watch requests for CRD resource at the node pool level.

Reply:
done